packages/integration-sdk-runtime/src/logger/index.ts

```diff
@@ -347,6 +347,10 @@ export class IntegrationLogger
         break;
       }
       case DisabledStepReason.PARENT_DISABLED: {
+        if (!additionalContext?.parentStep?.name) {
+          this.validationFailure(new Error('Parent step not provided'));
+          return;
+        }
         description += `Step was disabled because parent step "${additionalContext?.parentStep?.name}" was disabled. In order to enable this step, please check logs for more information about the parent step.`;
```

Comment:
let's skip the error and instead, if the parent step name is not included do:
`Step was disabled because parent step was disabled. In order to....
`